lib/samples/find_route/find_route_sample.dart

```diff
@@ -25,20 +25,31 @@ class FindRouteSample extends StatefulWidget {
 }
 
 class _FindRouteSampleState extends State<FindRouteSample> {
+  // the map view controller.
   final _mapViewController = ArcGISMapView.createController();
+  // the graphics overlay for the stops.
   final _stopsGraphicsOverlay = GraphicsOverlay();
+  // the graphics overlay for the route.
   final _routeGraphicsOverlay = GraphicsOverlay();
+  // the symbol for the route line.
   final _routeLineSymbol = SimpleLineSymbol(
       style: SimpleLineSymbolStyle.solid, color: Colors.blue, width: 5.0);
+  // the symbol for the start and end stops.
   final _routeStartSymbol = SimpleMarkerSymbol(
       style: SimpleMarkerSymbolStyle.circle, color: Colors.green, size: 8.0);
   final _routeEndSymbol = SimpleMarkerSymbol(
       style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 8.0);
+  // the stops for the route.
   final List<Stop> _stops = [];
```

Comment:
Since we're reviewing this sample, we can update some coding standard things. Here we would do `final _stops = <Stop>[];`. And then a couple lines below we would do `final _directions = <DirectionManeuver>[];`.